pom.xml

```diff
@@ -100,16 +100,28 @@
         </exclusion>
       </exclusions>
     </dependency>
-
     <!-- Needed for audit functionality -->
-    <!--
     <dependency>
```

Comment:
It's possible that I'm mis-reading the POM file...  they are still a bit of a mystery to me. :)